src/components/list-with-actions/_list-with-actions.scss

```diff
@@ -4,22 +4,42 @@
 
 .hmrc-list-with-actions {
   @include govuk-font($size: 19);
+
+  @include govuk-media-query($from: desktop) {
+    .hmrc-summary-list__key {
+      width: 50% !important;
+    }
+
+    .hmrc-summary-list__actions {
+      width: 50% !important;
+    }
```

Comment:
Ah, just realised you can't use override classes for these (in the template) as they're in a media-query so keeping these around is fine 👍🏻